codes/classical/bits/easy/checksum.yml

```diff
@@ -0,0 +1,28 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: checksum
+physical: bits
+logical: bits
+
+name: 'checksum code'
+#introduced: ''
+
+description: |
+  Checksum is an error detection code that makes use of dividing data into equal subunits of n bit lengths of up to 16-bit lengths. A checksum is generated utilizing the complement of the sum of n bits from the subunits and then appended to the end of the data unit. The process is reiterated on the receiver side, if the result is zero the data is error-free, if not the data is rejected. Checksum catches error bursts up to 16 bits but fails to catch transpositions and insertion/deletion of zeros.
+realizations:
+  - 'Used for error detection in internet protocols such as IP,TCP, and UDP'
+  - 'Typically strongers error detection algorithms, such as Cyclic Redundancy Check, are used first and checksum is the last check in end-to-end protocols \cite{doi:10.1016/C2010-0-65846-9}'
+relations:
+  parents:
+    - code_id: parity_check
+
+
+# Begin Entry Meta Information
+_meta:
+  # Change log - most recent first
+  changelog:
+    - user_id: GageErwin
+      date: '2023-8-11'
```

Comment:
I realize the error message is hard to extract from the pr check action script - the issue is that the date format is expected to be "2023-08-11" (month should have leading zero). I'm working on getting better error messages!